sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Authentication/IContainerRegistryAuthenticationClient.cs

```diff
@@ -8,10 +8,10 @@ namespace Azure.Containers.ContainerRegistry
 {
     internal interface IContainerRegistryAuthenticationClient
     {
-        Task<Response<AcrRefreshToken>> ExchangeAadAccessTokenForAcrRefreshTokenAsync(string service, string aadAccessToken, CancellationToken token = default);
-        Response<AcrRefreshToken> ExchangeAadAccessTokenForAcrRefreshToken(string service, string aadAccessToken, CancellationToken token = default);
+        Task<Response<AcrRefreshToken>> ExchangeAadAccessTokenForAcrRefreshTokenAsync(PostContentSchemaGrantType grantType, string service, string accessToken, CancellationToken cancellationToken = default);
+        Response<AcrRefreshToken> ExchangeAadAccessTokenForAcrRefreshToken(PostContentSchemaGrantType grantType, string service, string accessToken, CancellationToken cancellationToken = default);
```

Comment:
What motivates the change to this interface?  Why are we renaming the parameter names -- in ACR, it's important to distinguish between which accessToken is which because we use multiple: @Mohit-Chakraborty FYI

Reply:
Another side effect of the named enums. Parameters aren't renamed - `PostContentSchemaGrantType grantType` is added as a first parameter, cause `PostContentSchemaGrantType` is now an enum.